src/main/java/com/palantir/gradle/externalpublish/EnvironmentVariables.java

```diff
@@ -41,4 +41,8 @@ static boolean isTagBuild(Project project) {
                 .filter(tag -> !tag.isEmpty())
                 .isPresent();
     }
+
+    static boolean isFork(Project project) {
+        return envVarOrFromTestingProperty(project, "CIRCLE_PR_USERNAME").isPresent();
```

Comment:
From the docs for future reference ([ref](https://circleci.com/docs/2.0/env-vars/)):
> The GitHub or Bitbucket username of the user who created the pull request. **Only available on forked PRs.**